scipy/sparse/tests/test_sparsetools.py

```diff
@@ -61,6 +61,8 @@ def test_regression_std_vector_dtypes():
 
 
 @pytest.mark.slow
+@pytest.mark.skipif(not (sys.platform.startswith('linux') and np.dtype(np.intp).itemsize >= 8),
+                    reason="test requires 64-bit Linux")
```

Comment:
Can you use `@pytest.mark.xfail_on_32bit`?

Reply:
Done. I used the same message as with `TestConvolve2d.test_large_array`, I think it fits.